reverseCase/reverseCase.js

```diff
@@ -3,3 +3,18 @@
  * Example: 'Hello World' -> 'hELLO wORLD'
  * Assume that each string will contain only spaces and letters.
  */
+
+ const caseChanger = (str) => {
+     let transformed = "";
```

Comment:
Well done Sir! You're doing great. I think this is a very clean easy-to-read solution